src/pages/common/scanner_page.h

```diff
@@ -1,16 +1,17 @@
 #ifndef _SCANNER_PAGE_H_
 #define _SCANNER_PAGE_H_
 
-#define MIN_RADIOCHANNEL     0x00
-#define MAX_RADIOCHANNEL     0x4F
-
 struct scanner_page {
-    u8 channelnoise[MAX_RADIOCHANNEL - MIN_RADIOCHANNEL + 1];
+    u8 rssi[255];
```

Comment:
I don't like a magic number to define this. can we make it as a memory inside protocol? the other fields in this structure look like more options. let's split out them

Reply:
You mean 255? It's maximum scan channels for cc2500, cyrf only had 90. Rssi has to be written to by protocol and read out by scanner page, I don't think we can move this memory to protocol (thinking devo7e)